metadata.json

```diff
@@ -67,26 +67,26 @@
     "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.7.0"
+      "version_requirement": ">= 4.7.1 < 5.0.0"
```

Comment:
bump to < 6.0.0 plz

Reply:
just kidding

Reply:
tested it locally, doesnt work, reverted the commit. Fixing the issues for puppet5 should be handled in another PR.